### PR TITLE
Upgrade to symfony 3.4

### PR DESCRIPTION
**Pull request by ohvitorino**
This PR updates the project to Symfony 3.4 and fixes the deprecations.

### CODE REVIEW

**Comment by ohvitorino** [comment]
MultiUserBundle needs to be upgraded in order to get the deprecations fixed on travis.

File: composer.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "require": {
         "php": "^7.1",
 
-        "symfony/symfony": "^3.3",
+        "symfony/symfony": "^3.4",
         "symfony/framework-bundle": "^3.3",
```
Comment (jonasdekeukelaere) [inline composer.json]: shouldn't this be 3.4 too?

**Comment by jonasdekeukelaere** [comment]
Some more deprecations in tests

**Comment by jonasdekeukelaere** [comment]
MultiUserBundle PR has been merged.

File: composer.json (hunk 1 above)
Comment (ohvitorino) [reply composer.json]: Fixed on f5011c

**Comment by jonasdekeukelaere** [comment]
@ohvitorino Looks like the new MultiUserBundle version still needs to be releases and update in composer file, right?

**Comment by ohvitorino** [comment]
@jonasdekeukelaere Done.
I'm going to release this 😄

**Pull request merged**